packages/cozy-harvest-lib/.storybook/preview.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Preview } from "@storybook/react";
+
+import 'cozy-ui/dist/cozy-ui.min.css'
+import 'cozy-ui/dist/cozy-ui.utils.min.css'
+import 'cozy-ui/transpiled/react/stylesheet.css'
+import React from "react";
+import { StoryContainer } from "./StoryContainer";
+
+const preview: Preview = {
+  decorators: [
+    (Story) => (
+      <StoryContainer>
+        <div style={{position: "relative"}}>
+          <Story />
+        </div>
+      </StoryContainer>
+    ),
+  ],
+  parameters: {
```

Comment:
Should document/understand what those default parameters actually do, and if they are useful at all